component/button/button.css

```diff
@@ -0,0 +1,107 @@
+.btn {
+    display: inline-block;
+    font-size: 1rem;
+    padding:0.3125rem 1rem;
+    margin: 0.0625rem 0.125rem;
+    border-radius: 0.25rem;
+    text-align: center;
+    vertical-align: center;
+    border: 1px solid transparent;
+    cursor: pointer;
+    height: 2.25rem;
+}
+
+.btn-primary {
+    background-color: var(--primary-color);
+    color:white;
+}
+.btn-secondary {
+    background-color: var(--secondary-color);
+    color:white;  
+}
+.btn-link {
+    color:var(--primary-color);
+    background: white;
+    text-decoration: underline;
+}
+
+
+.btn-floating {
+    font-size: 3.125rem;
+    padding: 0.75rem;
+    cursor: pointer;
+    border-radius: 100%;
+    border: 1px solid transparent;
+    background-color: var(--primary-color);
+    color:white;
+    height: 1em;
+    width: 1.2em;
+    box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+    -webkit-box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+    -moz-box-shadow: 3px 4px 10px -2px rgba(0,0,0,0.75);
+}
```

Comment:
You can use variables for box shadow property above (DRY principle), it'll help you to be consistent also.

Reply:
yes understood. Thanks Ankur.